### ISSUE TITLE
Transpose radio button behaves differently when only chords are selected

### ISSUE THREAD

**Issue by zacjansheski** [musescore/MuseScore#23200]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

In the transpose window, with only chords selected, "To Key" cannot be selected, but you can click the radio button next to "By interval" and the radio button will look like "To Key" is selected

### Steps to reproduce

1. Select a chord
2. Tools>Transpose
3. Click radio button next to "To Key" it can't be selected
4. Click the radio button next to "By interval" Click it to switch the radio button

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/69917893/a3d58180-4feb-4781-949a-3f22508a472f

### MuseScore Version

MS4

### Regression

No.

### Operating system

All

### Additional context

 https://github.com/musescore/MuseScore/pull/23147 lets users transpose only chord symbols.
 
 If there is an issue with selecting "To Key" it should be completely greyed out.

**Comment by wizofaus** [comment]
"To Key" is supposed to be only available if a range is selected, I haven't looked to see whether it's *possible* to do "to key" if not.  There's a separate bug I've reported that if you select a range that has chord symbols AND notes in v2, but the notes are deselected (there's no way to deselect them if they're in v1), then transpose to key works for the chord symbols, but ALSO applies to the unselected notes, which seems wrong.

**Comment by wizofaus** [comment]
Yes, but I'm not sure why it would have anything to do with chord symbols. I assume you tested this on a build not including my fix to allow selecting "transpose chord names" regardless of whether it's a range?

**Comment by zacjansheski** [comment]
I just linked that PR to point out that it is, or will be functional.

I would like this issue to be focused on the UI aspect.  The current behavior feels janky.  If "To Key" is disabled it should be very clear that it is disabled.
